### PR TITLE
App Service - Postgen Modal

### PR DESCRIPTION
**Pull request by jsondoo**
**Changes**
- If app service is selected, it shows up on the post gen modal
- communication with extension happens through progressObject in #1014 

image

(our original plan was to not let users open project until app service is done, but we decided to just let them open it any time)

> If a user wants to use App Service, Functions, or Cosmos DB, they will be allowed to open the generated project at anytime, even if the resources haven't been deployed yet.
>  
> Reasoning: Users will want to edit their code first before doing any type of deployment. It is safe for them to poke through the code while this is happening.
>  
> Approach: Set expectations by letting users know how much time it will take to deploy each service (couple of minutes at worst)
>  
> 
> To follow this issue: https://github.com/microsoft/WebTemplateStudio/issues/1023

### CODE REVIEW

**Review by annielovesjs — APPROVED** [review]
looks great

**Review by Tanya0609 — APPROVED** [review]
works :) Good work!

File: src/client/src/containers/PostGenerationModal/index.tsx [hunk 1]
```diff
@@ -106,7 +112,7 @@ const PostGenerationModal = ({
     if (isTemplatesFailed) {
       return formatMessage(messages.restartWizard);
```
Comment (jcoc611-microsoft) [inline src/client/src/containers/PostGenerationModal/index.tsx]: Should this be moved up before the `return`?

**Review by jcoc611-microsoft — APPROVED** [review]
Nice!

**Pull request merged**